swift/StableDiffusion/tokenizer/T5Tokenizer.swift

```diff
@@ -9,11 +9,11 @@ import Tokenizers
 public extension Config {
     /// Assumes the file is already present at local url.
     /// `fileURL` is a complete local file path for the given model
-    public init(fileURL: URL) throws  {
+    init(fileURL: URL) throws  {
         let data = try Data(contentsOf: fileURL)
         let parsed = try JSONSerialization.jsonObject(with: data, options: [])
-        guard var dictionary = parsed as? [String: Any] else { throw Hub.HubClientError.parse }
-        
+        guard var dictionary = parsed as? [NSString: Any] else { throw Hub.HubClientError.parse }
```

Comment:
I don't think this is necessary. Can you explain?

Reply:
Yes, because it is part of here: 

https://github.com/huggingface/swift-transformers/blob/313fbd7305fee3919012b68d02443fa620325dbc/Sources/Hub/Hub.swift#L43